### PR TITLE
chore: update readme, update parameters, add validation

### PR DESCRIPTION
**Pull request by denis-kralj-novu**

### CODE REVIEW

File: CODEOWNERS
Comment (denis-kralj-novu) [inline CODEOWNERS:1]: removed as per recommendation of original readme

File: badges/coverage.svg
Comment (denis-kralj-novu) [inline badges/coverage.svg:1]: hardcoded 100% coverage on a project with no tests, removed

File: package.json [hunk 1]
```diff
@@ -1,21 +1,21 @@
 {
-  "name": "@novuhq/bridge-sync-action",
+  "name": "@novuhq/novu-sync-action",
   "description": "GitHub Actions TypeScript template",
-  "version": "0.0.0",
+  "version": "0.0.4",
```
Comment (denis-kralj-novu) [inline package.json]: @rifont I've aligned this with the version we have in the sync modal, let me know if it should be something else

Comment (rifont) [reply package.json]: Let's publish this as version `2.0.0` to reflect the v2 release, so we'll update both here and the GH action version via GH releases

File: README.md [hunk 2]
```diff
@@ -1,212 +1,36 @@
...
-
-After testing, you can create version tag(s) that developers can use to
-reference different stable versions of your action. For more information, see
-[Versioning](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md)
-in the GitHub Actions toolkit.
-
-To include the action in a workflow in another repository, you can use the
-`uses` syntax with the `@` symbol to reference a specific branch, tag, or commit
-hash.
+## Using the Action in your CI
```
Comment (rifont) [inline README.md]: Let's move this to be the first section, it's what the consumers are after

File: action.yml [hunk 3]
```diff
@@ -9,14 +9,14 @@ branding:
...
-    description: 'Your Novu API key'
+  secret-key:
+    description: 'Your Novu secret key'
     required: true
   bridge-url:
     description: 'Your Bridge deployment endpoint URL'
     required: true
   api-url:
-    description: 'Your Novu cloud API base URL'
+    description: 'The Novu Cloud API base URL'
```
Comment (rifont) [inline action.yml]: For consistency in CLI naming.
```suggestion
    description: 'The Novu Cloud API URL'
```

**Review by rifont — APPROVED** [review]
🦾

File: README.md (hunk 2 above)
Comment (denis-kralj-novu) [reply README.md]: I will try to follow the format **[of this action](https://github.com/actions/checkout)**, as it is well established

**Pull request merged**